src/theme/components/notification.ts

```diff
@@ -121,6 +122,7 @@ const NotificationContent = {
             color: "dark.ui.link.secondary",
           },
         },
+        _focus: { ...customFocusColor("ui.black", "dark.ui.link.primary") },
```

Comment:
Use `dark.ui.typography.body` for dark mode.  The dark mode link color does work, but, for consistency, I would like to use the same color as the text.